plugin-desktopswitch/desktopswitch.cpp

```diff
@@ -293,11 +254,11 @@ void DesktopSwitch::settingsChanged()
         int columns = static_cast<int>(ceil(static_cast<float>(m_desktopCount) / mRows));
         if (panel()->isHorizontal())
         {
-            mDesktops->setDesktopLayout(NET::OrientationHorizontal, columns, mRows, mWidget.isRightToLeft() ? NET::DesktopLayoutCornerTopRight : NET::DesktopLayoutCornerTopLeft);
+            //mDesktops->setDesktopLayout(NET::OrientationHorizontal, columns, mRows, mWidget.isRightToLeft() ? NET::DesktopLayoutCornerTopRight : NET::DesktopLayoutCornerTopLeft);
         }
         else
         {
-            mDesktops->setDesktopLayout(NET::OrientationHorizontal, mRows, columns, mWidget.isRightToLeft() ? NET::DesktopLayoutCornerTopRight : NET::DesktopLayoutCornerTopLeft);
+            //mDesktops->setDesktopLayout(NET::OrientationHorizontal, mRows, columns, mWidget.isRightToLeft() ? NET::DesktopLayoutCornerTopRight : NET::DesktopLayoutCornerTopLeft);
         }
```

Comment:
Why is `NETRootInfo::setDesktopLayout()` removed under X11? Isn't this a regression with X11?

Reply:
Hi, I didn't understand what that code is about. However it should go to backend now.

Reply:
> it should go to backend now.

Yes, and it should be done here, in this PR.

Reply:
Sorry for my mistake! The PR is big, and GitHub didn't load a file. I cloned your branch now, to read it without GitHub's hassles.

Reply:
> I didn't understand what that code is abou

See https://api.kde.org/frameworks/kwindowsystem/html/classNETRootInfo.html#a06f991c0b1540e779e5e324404337241

If I'm not mistaken again, this is really removed in your PR: the desktop layout isn't set according to the pager anymore.